po/superdesk.pot

```diff
@@ -11517,72 +12609,92 @@ msgstr ""
 msgid "{{ name }} is too short"
 msgstr ""
 
-#: ../superdesk-planning/client/actions/assignments/notifications.js:318
+#: scripts/apps/monitoring/views/aggregate-settings.html:109
+#: scripts/apps/monitoring/views/aggregate-settings.html:88
+msgid "{{ output.listLabel }}"
+msgstr ""
+
+#: scripts/apps/monitoring/views/monitoring-view.html:23
+msgid ""
+"{{ profile.label }}\n"
+"                                        <i ng-show=\"aggregate.activeFilters.contentProfile.includes(profile._id)\" class=\"icon-ok\"></i>"
```

Comment:
looks like there is some extra `translate`

Reply:
I'm not sure what you mean

Reply:
I mean https://github.com/pablopunk/superdesk-client-core/blob/7c080e95badc1aaa6d77cb95d762b2baed01d94c/scripts/apps/monitoring/views/monitoring-view.html#L26-L28 there it `translate>` which makes this whole thing including `<i ng-show...` being string for translation on transifex

Reply:
ok I can make another PR with a `<span translate>` to get only the label

Reply:
https://github.com/superdesk/superdesk-client-core/pull/3391